go/adapter/otel_stdout/adapter.go

```diff
@@ -4,96 +4,94 @@ import (
 	"encoding/json"
 	"fmt"
 	"log"
-	"time"
 
 	observe "github.com/dylibso/observe-sdk/go"
+	"github.com/dylibso/observe-sdk/go/adapter/otel_formatter"
 	otel "github.com/dylibso/observe-sdk/go/adapter/otel_formatter"
 )
 
 type OtelStdoutAdapter struct {
 	observe.AdapterBase
-	TraceId string
 }
 
 func NewOtelStdoutAdapter() OtelStdoutAdapter {
 	base := observe.NewAdapterBase()
 	return OtelStdoutAdapter{
 		AdapterBase: base,
-		TraceId:     observe.NewTraceId().ToHex16(),
 	}
 }
+func (o *OtelStdoutAdapter) HandleTraceEvent(te observe.TraceEvent) {
+	if te.TelemetryId == nil {
+		log.Println("Otel adapter needs a trace id")
```

Comment:
similar to the question about the Datadog adapter -- should we generate a ID here if the user hasn't supplied it, or should that already be done earlier?

Reply:
yes, it is set upstream but i went and and changed it from a pointer to a value type as the property